pkg/k8sutil/k8sutil.go

```diff
@@ -247,6 +249,18 @@ func SanitizeVolumeName(name string) string {
 	return strings.Trim(name, "-")
 }
 
+// HostAliasesMonitoringV1ToCoreV1 converts array of monitoringv1 HostAlias to array of corev1 HostAlias
+func HostAliasesMonitoringV1ToCoreV1(input []monitoringv1.HostAlias) []v1.HostAlias {
```

Comment:
For consistency, it can be moved to `pkg/operator/types.go` and renamed `MakeHostAliases`